modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -0,0 +1,430 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
+
+The Salesforce platform uses the System Administrator profile which contains permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile or the AEH_Anypoint permission set is required for the user who performs the Salesforce linking.
+
+
+== Before You Begin
+Consider the permission strategy for users in the Anypoint organization and Salesforce organization.
+
+== Grant the API Experience Hub Administrator Permission
+The Organization Administration and API Experience Hub Administrator can perform all of the functions to create a portal. Only the Organization Administrator or an existing API Experience Hub Administrator can grant a user the API Experience Hub Administrator permission. Grant this permission when you want another user in your organization to perform the Salesforce linking process, create and curate the portal, manage the portal, and manage users.
```

Comment:
Is "Organization Administration" correct or should it be [Administrator](https://docs.mulesoft.com/access-management/permissions-by-product#:~:text=Access%20Management-,Organization%20Administrator,-At%20the%20root)?

Reply:
[API Experience Hub Admin](API Experience Hub Admin)?

Reply:
If ☝️ correct, apply globally.

Reply:
Organization Administrator is correct and so is API Experience Hub Administrator.